src/DotNetProjectFile.Analyzers/Rule.cs

```diff
@@ -128,6 +128,15 @@ public static class Rule
         category: Category.CodeQuality,
         isEnabled: true);
 
+    public static DiagnosticDescriptor UseSonarAnalyzers => New(
```

Comment:
If this is 'for packages', should it have the ForPackages suffix just like the rule above?